src/content/quickstart/vsphere_aws/_index.md

```diff
@@ -0,0 +1,170 @@
+
+{{% notice tip %}}
+
+Although we are using default OCP network configuration on vSphere, you can install vSphere with custom network customization as shown [here](https://red.ht/2WFjEVg)
+
+{{% /notice %}}
+
+### Deploy Cluster on AWS
+
+#### Configure AWS CLI with appropriate values
+
+```bash
+$ aws configure
+AWS Access Key ID [None]: ....
+AWS Secret Access Key [None]: ....
+Default region name [None]: ....
+Default output format [None]: text
+```
+
+For more details, follow this link:
+
+> https://docs.openshift.com/container-platform/4.4/installing/installing_aws/installing-aws-account.html
+
+In this step we shall deploy cluster-b on AWS, modifying the default Cluster and Service CIDRs
+
+| Pod CIDR     | Service CIDR |
+|--------------|--------------|
+|10.132.0.0/14 |172.31.0.0/16 |
+
+
+```bash
+openshift-install create install-config --dir cluster-b
+```
+
+Change the POD IP network, please note it’s a /14 range by default so you need to use 
++4 increments for “128”, for example: 10.132.0.0, 10.136.0.0, 10.140.0.0, ...
+ 
+```bash
+sed -i 's/10.128.0.0/10.132.0.0/g' cluster-b/install-config.yaml
+```
+
+Change the service IP network, this is a /16 range by default, so just use +1 increments
+for “30”: for example: 172.31.0.0, 172.32.0.0, 172.33.0.0, ...
+
+```bash
+sed -i 's/172.30.0.0/172.31.0.0/g' cluster-b/install-config.yaml
+```
+
+And finally deploy
+
+```bash
+openshift-install create cluster --dir cluster-b
+```
+
+### Make your AWS cluster ready for Submariner
+
+Submariner gateway nodes need to be able to accept IPsec traffic. The default ports are 4500/UDP and 500/UDP.
+However, when you have some clusters in OnPrem (like vSphere in this example) which are typically inside a corporate network, the firewall configuration on the corporate router may not allow the default IPsec traffic.
+We can overcome this limitation by using non-standard ports like 4501/UDP and 501/UDP.
+
+Additionally, the default OpenShift deployments do not allow assigning an elastic public IP
+to existing worker nodes, something that's necessary at least on one end of the IPsec connections. 
+
+To handle these requirements on AWS, we provide a script that will prepare your AWS OpenShift deployment
+for submariner, and will create an additional gateway node with an external IP.
+
+```bash
+
+curl https://raw.githubusercontent.com/submariner-io/submariner/master/tools/openshift/ocp-ipi-aws/prep_for_subm.sh -L -O
+chmod a+x ./prep_for_subm.sh
+
+./prep_for_subm.sh cluster-b      # respond yes when terraform asks
+
+```
+
+> **_INFO_** Please note that  **oc**, **aws-cli**, **terraform**, and **unzip** need to be installed before running the `prep_for_subm.sh` script.
+
+Currently, prep_for_subm.sh script does not support specifying custom IPsec ports.
+Until the [issue](https://github.com/submariner-io/submariner/issues/240) is resolved, execute the following commands to open the necessary ports
```

Comment:
We'll have to remember to update this section once resolved 😄

Reply:
haha...

Added a note on the issue to update this section after we fix it - https://github.com/submariner-io/submariner/issues/240#issuecomment-625658031